### ISSUE TITLE
DACs to automatically implement the results of the min/max test.

### ISSUE THREAD

**Issue by cgourcuf** [OneArgo/ADMT#8]
- [ ] AOML
- [ ] BODC
- [ ] Coriolis
- [ ] CSIO
- [ ] CSIRO
- [ ] INCOIS
- [ ] JMA
- [ ] KMA
- [ ] KORDI
- [ ] MEDS
- [ ] NMDIS

**Comment by jlovellcsiro** [comment]
CSIRO have not implemented this. We try and act promptly on alerts from Coriolis.

**Comment by Argo-JMA** [comment]
Unfortunately, JMA doesn't have human resources to introduce the min/max test.

**Comment by liuzenghong** [comment]
CSIO has implemented a MinMax test developed by IAP/CAS in our global Argo T/S post-qc system. I think we can also implemented this in the RTQC.

**Comment by cgourcuf** [comment]
Feedback will be presented & discussed at AMDT26

**Comment by cgourcuf** [comment]
Agreed at ADMT26 that further work is needed before DACs can implement the test in RT. A WG was created to work on a more apropriate version of the test (https://github.com/OneArgo/ADMT/issues/197).
